src/components/SearchComponent/index.js

```diff
@@ -58,36 +59,46 @@ const SearchComponent = ({ setCurrentCards, teamInformation }) => {
     setCityName(e.target.value);
   };
   return (
-    <form onSubmit={handleOnSubmit}>
-      <div className="searchClass">
+    <div className="m-5 p-5 bg-purple-50">
+      <form onSubmit={handleOnSubmit}>
         <input
           type="text"
+          placeholder="github user"
           value={nameOrDesignation}
           onChange={(event) => {
             setNameOrDesignation(event.target.value);
             setCityName("");
           }}
         />
-        <button className="buttonClass">Search</button>
-      </div>
-
-      <select className="stateClass" value={state} onChange={handleStateChange}>
-        {Object.keys(StateCityInformation).map((StateName) => (
-          <option key={StateName} value={StateName}>
-            {StateName}
-          </option>
-        ))}
-      </select>
-
-      <select value={cityName} onChange={handleCityChange}>
-        {city &&
-          city.map((cities) => (
-            <option key={cities} value={cities}>
-              {cities}
+        <select value={state} onChange={handleStateChange}>
+          {Object.keys(StateCityInformation).map((StateName) => (
+            <option key={StateName} value={StateName}>
+              {StateName}
             </option>
           ))}
-      </select>
-    </form>
+        </select>
+
+        <select value={cityName} onChange={handleCityChange}>
+          {city &&
+            city.map((cities) => (
+              <option key={cities} value={cities}>
+                {cities}
+              </option>
+            ))}
+        </select>
+        <button className="bg-blue-700 h-12 w-28 font-bold m-5 rounded-xl text-white">
+          Search
+        </button>
+        <button
+          className={`h-12 w-44 font-bold m-5 rounded-xl border-black border-spacing-0  text-pink-500 ${
+            color === "purple" ? "bg-purple-50" : "bg-black"
+          }`}
+          onClick={() => setColor(color === "purple" ? "black" : "purple")}
+        >
```

Comment:
can move this function [setColor] out of button tag.

Reply:
moved